spec/fixtures/documentation/permissions.yml

```diff
@@ -4,3 +4,23 @@ user_comedor:
   subject: super_admin (User)
   object: comedor (Documentation::Workspace)
   action: full
+
+read_access_user:
+  subject: read_user (User)
+  object: comedor (Documentation::Workspace)
+  action: read
+
```

Comment:
I think we should use factories instead, before it gets crowded with fixtures

Reply:
Same for the users

Reply:
@alexisKinto these are really simple test, I don't see that we might use factories at this test... When the test are kind of complicated, I really understand, but for this is a simple test.

Reply:
Agree, but it will be easier to make this changes now than later

Reply:
I agree to leave this in fixtures because permissions and users are relatively static and aren't temporal. (don't depend on a date)